index.html

```diff
@@ -50,44 +50,175 @@
+              where people engage with your content helps inform better
+              decisions.
+            </p>
+          </div>
+          <!-- card 3-->
+
+          <div class="card">
+            <div class="image">
+              <img src="./images/icon-fully-customizable.svg" alt="" />
+            </div>
+            <h2>Fully Customizable</h2>
+            <p>
+              Improve brand awareness and content discoverability through
+              customizable links, supercharging audience engagement.
+            </p>
+          </div>
+        </div>
       </div>
     </div>
 
-    <div class="started">
+    <div class="boost">
       <!-- Aquí va septima seccion-->
-      Boost your links today Get Started
+      <div class="container"> 
+        <div class="boostContent"> 
+          <div id="boostLabelDiv"> 
+            <label for="boost" class="boostLabel">
+              Boost your links today
+            </label> 
+          </div> 
+          <div id="getBoostdivider"> 
+            <a class="getBoost" href="#">
+              Get Started
+            </a> 
+          </div> 
+        </div> 
+      </div>
     </div>
-    <footer class="footer">
-      <!-- Aquí va sexta seccion footer-->
-      Features Link Shortening Branded Links Analytics Resources Blog Developers
-      Support Company About Our Team Careers Contact
-    </footer>
+<footer class="footer">
+    <!-- Aquí va sexta seccion footer-->
+    <div class="container">
+      <div class="footerContent">
+        <div class="imgFooterContent">
+          <img src="./images/logo.svg" alt="" class="footerLogoFilter" />
+        </div>
+        <div class="featuresFooter">
+          <h1 for="Features" class="footerLabelTitle">Features</h1>
+          <label for="LinkShortening">Link Shortening</label>
+          <label for="BrandedLinks">Branded Links</label>
+          <label for="Analytics">Analytics</label>
+        </div>
+        <div class="resourcesFooter">
+          <h1 for="Resources" class="footerLabelTitle">Resources</h1>
+          <label for="Blog">Blog</label>
+          <label for="Developers">Developers</label>
+          <label for="Support">Support</label>
+        </div>
+
+        <div class="companyFooter">
+          <h1 for="Company" class="footerLabelTitle">Company</h1>
+          <label for="About">About</label>
+          <label for="Our Team">Our Team</label>
+          <label for="Careers">Careers</label>
+          <label for="Contact">Contact</label>
+        </div>
+
+        <div class="socialNetworksFooter">
+          <label><i><img src="./images/icon-facebook.svg" alt=""></i></label>
+          <label><i><img src="./images/icon-twitter.svg" alt=""></i></label>
+          <label><i><img src="./images/icon-pinterest.svg" alt=""></i></label>
+          <label><i><img src="./images/icon-instagram.svg" alt=""></i></label>
+        </div>
+      </div>
+    </div>
+  </footer>
```

Comment:
Perfect. Yet we need to set the css file for a mobile display in navbar and footer sections, when the resolution is lower than 950px.